mccodec/mc/configuration/clientbound_registry_data.mbt

```diff
@@ -0,0 +1,36 @@
+///|
+/// TODO: Definition of entries
+/// Registry Data packet (0x07)
+/// Represents registries sent from server to client
+struct RegistryDataS2C {
+  registry_id : String // Registry identifier, TODO: Identifier type
+  entries : FixedArray[(String, String)] // Entry ID and NBT data pairs
```

Comment:
Although I mentioned earlier this could be a map, I couldn't prove my point this time since Notchian client is likely to store these entries in List (Refer to SimpleRegistry). You should take care of this as this may contain duplicated entries.